src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_1909.ps1

```diff
@@ -30,7 +36,4 @@ use multiple lines to tell you how super secure it is.
 }
 
 Win10_1909_L1
-#Uncomment the following line to also apply the configuration on the localmachine
-#Start-DscConfiguration -Path '.\Win10_1909_L1' -Verbose -Wait -Force
-#This version will capture errors and verbose to a log file
-#Start-DscConfiguration -Path '.\Win10_1909_L1' -Verbose -Wait -Force 4>&1 2>&1 > 'c:\DSC.log'
\ No newline at end of file
+Start-DscConfiguration -Path '.\Win10_1909_L1' -Verbose -Wait -Force
```

Comment:
Pausing for reflection on having a command with -Force un-commented here. Are we confident this will result in the desired behavior for anyone loading this module?

Reply:
This appears in other places, but I'll just comment once. I'm not speaking specifically to this module - just asking about the pattern change.

Reply:
-Force in this context will just tell the LCM to drop what it's doing and do what you're asking it do to here. I can see an argument for removing it here.